crates/cargo-winrt/src/main.rs

```diff
@@ -67,14 +67,32 @@ fn parse_args() -> Result<Subcommand, ArgsError> {
         subcommand = args.subcommand()?;
     }
 
-    let verbose = args.contains(["-v", "--verbose"]);
-    let force = args.contains(["-f", "--force"]);
-    args.finish()?;
-
     let subcommand = match subcommand.as_deref() {
-        Some("install") => Subcommand::Install(Install { verbose, force }),
-        Some("run") => Subcommand::Run(Run { verbose, force }),
-        Some("build") => Subcommand::Build(Build { verbose, force }),
+        Some("install") => {
```

Comment:
This seems very repetitive. Is it worth cleaning up?

Reply:
Do you mean just the duplicated argument parsing? My thought was that although they're currently all the same, they wouldn't  necessarily *always* be, so splitting it up by command seemed sensible. Originally they were always parsed, but since, for example, the `help` subcommand doesn't use those arguments, it needed to be rethought in some way.

It's also possible that, for example, we'd want to get crazier with the `run` and `build` subcommands that forward to Cargo and let you pass additional arguments that get forwarded on.

So that's kind of a lot of words to say, this could probably be cleaned up (for example, we could move parsing into constructors for each subcommand so at least it wasn't just repetitive here), but it wasn't clear to me what the most straightforward path for potential future evolution was here, and this seemed at least simplest.